src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/subgraph_select_pd.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/node_builders/constant.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "utils/cpu_test_utils.hpp"
+
+namespace ov {
+namespace test {
+
+/*
+    input1(f32_abcd_{1,64,32,32})                      input2(f16_abcd_{1,128,1,1})
+        |                                                 |
+    Reorder(f32_acdb_{1,64,32,32})   const             Convert(f32_abcd_{1,128,1,1})
+        |                           /                     |
+        |                          /                      |
+    Convolution(f32_acdb_{1,1,30,30})  Range_1520      VariadicSplit(f32_abcd_{1,64,1,1}, f32_abcd_{1,64,1,1})
+        |                             /                   \                               /
+        |                           /                      \                             /
+        |                         /                         \                           /
+        |                       /                            \                         /
+    MVN(f32_acdb_{1,1,30,30})              Reorder1(f32_acdb_{1,64,1,1})  Reorder2(f32_acdb_{1,64,1,1})
+            \                             /                            /
+             \                           /                            /
+              \                         /                            /
+               \                       /                            /
+               Subgraph(f32_acdb_{1,64,30,30})
+                            |
+                            |
+               Convolution(f32_acdb_{1,1,28,28})
+                            |
+                          Result
+    The Subgraph node have 3 inputs: they don't have same layout.
+    Expected: Reorder is inserted after VariadicSplit[0] and VariadicSplit[1], not inserted after MVN.
+    Because VariadicSplit's output layout is [1,64,1,1], it' reorder almost have no calculation.
```

Comment:
Is it really about "almost no calculation" rather than less compute in comparison with other Reorder insertion options?

Reply:
Function calls themselves take time, and sequential copying also takes time, so perhaps "less computation" is more accurate.